modules/hostap/src/supp_events.c

```diff
@@ -9,10 +9,158 @@
 #include "includes.h"
 #include "common.h"
 
-int send_wifi_mgmt_event(const char *ifname, enum net_event_wifi_cmd event, int status)
+#define MAC_STR_FORMAT "%2hhx:%2hhx:%2hhx:%2hhx:%2hhx:%2hhx"
+
+char *event_map[] = {
```

Comment:
static and also better to use proper namespaces i.e., prefix here

Reply:
Done